packages/esm-ohri-pmtct-app/src/views/mch-summary/tabs/current-pregnancy.component.tsx

```diff
@@ -161,7 +163,8 @@ const CurrentPregnancy: React.FC<PatientChartProps> = ({ patientUuid }) => {
         </Link>
       );
       let relativeObject: familyItemProps = {
-        id: relativeToIdentifierMap.find((entry) => entry.patientId === relative.personB.uuid)?.pTrackerId,
+        id: i,
```

Comment:
i thought we was using the uuid? there's a danger with using interating numbers with such lists